src/components/main/main.js

```diff
@@ -18,7 +18,7 @@ function Main() {
         <Route exact path="/" component={Products} />
         <Route exact path="/admin-orders" component={AdminOrders} />
```

Comment:
It may be a bit late in the game to recommend this, but it might be a bit safer to put all of your router paths in an object exported from a constants file. I see that you reference those string in other parts of the code. If there is a case where something has to change, you will have to look for every reference to make that change. Doing this now will make you much happier in that hypothetical future